src/images/twitter.svg

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
This is now in cloudinary. Please remove from git

Reply:
Done